packages/fxa-content-server/app/styles/modules/_avatar.scss

```diff
@@ -35,128 +35,15 @@
   position: relative;
   width: $avatar-size;
 
-  .avatar-camera & {
-    margin-bottom: 20px;
+  &.spinner-completed {
+    border: 2px solid $avatar-border-color;
+    box-shadow: $card-box-low;
   }
 
-  &.card-view {
```

Comment:
I removed some stuff we're not using anymore and removed other cruft. I was going to make a separate ticket to TW-ify some of these bits but we'd also need to do that for the `avatar_spinner.scss` file... seems best to just do that when we convert to React, I left a comment in `images.css` about it.